gitrevise/todo.py

```diff
@@ -166,6 +166,7 @@ def edit_todos_msgedit(repo: Repository, todos: List[Step]) -> List[Step]:
 
         However, if you remove everything, these changes will be aborted.
         """,
+        type_=BufferType.REVISE_TODO_MSG_EDIT,
```

Comment:
This `++ pick 0f123` format does not fit into existing syntax highlighting and `syntax_hint` returns nothing, maybe switch to `pick +++` to make this work?

Reply:
This would be a fairly large change to the UI of the `-ie` mode of git-revise, which I would generally prefer not to make.